index.html

```diff
@@ -39,315 +37,319 @@
-                        3 Strand Code will put you through the Agile ropes executing your own ideas.
-                        You'll be prepared to kick off your own projects, or help bring best practices to established.
-                    </p>
-                    <br class="visible-xs"/>
-                    <br class="visible-xs"/>
-                </div>
+  <div class="container">
+    <div class="bullets">
+      <div class="row text-center">
+        <div class="col-xs-10 col-xs-offset-1 col-sm-offset-0 col-sm-4">
+          <div class="icon">
+            <i class="fa fa-lightbulb-o"></i>
+          </div>
+          <h3>
+            Project Pitches
+          </h3>
+
+          <p>
+            Pitch your own idea or join up with someone else's.
+            You'll spend the course helping bring it to life with peers and top mentors.
+            It's a sure fire path to learning fast and having a blast.
+          </p>
+          <br class="visible-xs"/>
+          <br class="visible-xs"/>
+        </div>
+        <div class="col-xs-10 col-xs-offset-1 col-sm-offset-0 col-sm-4">
+          <div class="icon">
+            <i class="fa fa-flash"></i>
+          </div>
+          <h3>
+            Agile SCRUM
+          </h3>
+
+          <p>
+            3 Strand Code will put you through the Agile ropes executing your own ideas.
+            You'll be prepared to kick off your own projects, or help bring best practices to established.
+          </p>
+          <br class="visible-xs"/>
+          <br class="visible-xs"/>
+        </div>
 
-                <div class="col-xs-10 col-xs-offset-1 col-sm-offset-0 col-sm-4">
-                    <div class="icon">
-                        <i class="fa fa-comments-o"></i>
-                    </div>
-                    <h3>
-                        Collaboration
-                    </h3>
-
-                    <p>
-                        Our mentors and leaders will work with you hands on and face-to-face.
-                        You'll also work with a group of peers throughout the course.
-                        The relationships and network you gain here will last forever.
-                    </p>
-                </div>
-            </div>
+        <div class="col-xs-10 col-xs-offset-1 col-sm-offset-0 col-sm-4">
+          <div class="icon">
+            <i class="fa fa-comments-o"></i>
+          </div>
+          <h3>
+            Collaboration
+          </h3>
+
+          <p>
+            Our mentors and leaders will work with you hands on and face-to-face.
+            You'll also work with a group of peers throughout the course.
+            The relationships and network you gain here will last forever.
+          </p>
         </div>
+      </div>
     </div>
+  </div>
 </section>
 
 <!-- Pricing and Registration-->
 <div class="pricing-registration">
-    <div class="container">
-
-        <div id="pricing" class="tsc-scroll-anchor"></div>
-        <section ng-if="false">
```

Comment:
This was hiding pricing, though the registration form still showed the price.  Moved to only hide the seating chart, but still show pricing and discount.